src/macports1.0/macports.tcl

```diff
@@ -654,8 +654,7 @@ proc mportinit {{up_ui_options {}} {up_options {}} {up_variations {}}} {
         macports::macosx_deployment_target \
         macports::archivefetch_pubkeys \
         macports::ping_cache \
-        macports::host_blacklisted \
-        macports::host_preferred \
+        macports::host_cache \
```

Comment:
`host_cache` does not seem to be used in mportinit, or did I miss it?

`host_backlist` and `preferred_hosts`, however, are – although I guess technically it was never necessary to add them to this global statement anyway?

Reply:
Technically not necessary here, but I thought I should list it for completeness.